runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -110,11 +111,15 @@ public Timestamp getTimestamp() {
                              @Nullable final Class<M> mClass,
                              @Nonnull final TableOptions tableOptions)
             throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        long startTime = System.nanoTime();
         Optional<Timer.Sample> sample = MeterRegistryProvider.getInstance().map(Timer::start);
         Table table =
                 runtime.getTableRegistry().openTable(namespace, tableName, kClass, vClass, mClass, tableOptions);
         corfuStoreMetrics.recordTableCount();
+        int tableSize = table.count();
         table.getMetrics().recordTableOpenTime(sample);
+        log.info("openTable {}${} took {}ms to load {} entries", namespace, tableName,
+                TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime), tableSize);
```

Comment:
If you are logging in milliseconds why do you need to account for time in nanoseconds? converting from nanoseconds to milliseconds will lose any values less than 1ms, so the conversion is superfluous. 

I suggest you just use milliseconds, or instead use microseconds.

Reply:
This is just a helper routine to convert the units, nanoseconds tends to be produce long lines, so just picked milliseconds.
The api I use to measure time is System.nanoTime() is there another alternative I should be using?

Reply:
You can use `System.currentTimeMillis()` , if the load time is less than 1ms and you still want to see how long it takes you can use `System.nanoTime()` and convert to microseconds.

Reply:
Done.